Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerForMainServer.java

```diff
@@ -27,8 +28,13 @@ public boolean createSession(String name, String password, String hostName, int
     }
 
     @Override
-    public int getAvailableMemory() throws RemoteException {
-        return (int) Runtime.getRuntime().freeMemory();
+    public long getAvailableMemory() throws RemoteException {
+        return Runtime.getRuntime().freeMemory();
+    }
+
+    @Override
+    public int ping(int value) throws RemoteException {
+        return value * RmiConstants.PING_CALCULATION_FACTOR;
     }
 
     public final void close() throws IOException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)